proxy_await.c

```diff
@@ -286,6 +286,16 @@ int mcplib_await_return(io_pending_proxy_t *p) {
                         valid = false;
                     }
                     break;
+                case AWAIT_FASTGOOD:
+                    if (p->client_resp->status == MCMC_OK) {
+                        // End early on a hit.
+                        if (p->client_resp->resp.code != MCMC_CODE_END) {
+                            aw->wait_for = 0;
+                        } else {
+                            is_good = true;
+                        }
+                    }
+                    break;
```

Comment:
I think you need to also whitelist AWAIT_FASTGOOD in mcplib_await() (around L49) to have this working.

Reply:
fixed, thanks!